### PR TITLE
First contribution

### PR DESCRIPTION
**Pull request by EONRaider**
A lot was done to the files, but not much was modified in terms of functionality. Most modifications are related to code quality and formatting.

The one major difference is the removal of the .conf file. When dealing with credentials, when it comes to best-practices, you should pass all the necessary information as environment variables to the application. I did not modify the instructions on the README.md file yet, but it can be done in the future. This implementation should make things easier. The necessary variables are WEBULL_USER and WEBULL_PASS. This will serve well in the future when we use Docker.

Also, could you create a dev branch just so I can avoid pushing new PRs to main in the future?

### CODE REVIEW

**Comment by mlebjerg** [comment]
Love this! You really cleaned up the mess!
I am not entirely ready to give up the .conf file yet as it is an easy way to set these two things and one of the standard practice for Docker. Would you mind bringing this back for the time being? Then this can go into main!

I will make a dev branch after this comment

File: mm_graph.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+              "filter?selectedDate={}&market=&filterToken=XRT&max=10&offset=0" \
+              "&pageNumber=1&sortOrder=up&sortType="
+
+data = yf.download(
+    # tickers list or string as well
+    tickers=symbol,
+
+    # use "period" instead of start/end
+    # valid periods: 1d,5d,1mo,3mo,6mo,1y,2y,5y,10y,ytd,max
+    # (optional, default is '1mo')
```
Comment (peacefulchickenk) [inline mm_graph.py:17]: Should this say `# (optional, default is '1d')`?

File: mm_graph.py [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    period="1d",
+
+    # fetch data by interval (including intraday if period < 60 days)
+    # valid intervals: 1m,2m,5m,15m,30m,60m,90m,1h,1d,5d,1wk,1mo,3mo
+    # (optional, default is '1d')
```
Comment (peacefulchickenk) [inline mm_graph.py:22]: Should this say `# (optional, default is '1m')`?

File: mm_signals.py [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+            console.print(f"{base_str} NEWS PENDING.")
+            create_deal(deal)
+
+        if deal["volume"] == "1000":
+            console.print(f"{base_str} Don't let it run!")
+            create_deal(deal)
+
+        if deal["volume"] == "2100":
+            console.print(f"{base_str} Let it run!")
+            create_deal(deal)
```
Comment (peacefulchickenk) [inline mm_signals.py:117]: I rolled this up in this manner, would you like to use this instead?

```py
        volumes = {
            "100": { "msg": "I need Shares", "create_deal": False },
            "200": { "msg": "I need Shares badly do not take stock down", "create_deal": False },
            "300": { "msg": "Take/Taking the stock down so I can load shares", "create_deal": True },
            "400": { "msg": "Keep trading it sideways", "create_deal": True },
            "500": { "msg": "Gap the stock ", "create_deal": True, "direction": { "B": "up ⬆️", "G": "up ⬆️", "S": "down ⬇️", "L": "down ⬇️" } },
            "505": { "msg": "I am short on shares", "create_deal": True },
            "600": { "msg": "Apply resistance at ", "create_deal": True, "suffix": "price" },
            "700": { "msg": "Move the price up", "create_deal": True },
            "777": { "msg": "Move the price up", "create_deal": True },
            "800": { "msg": "Volume coming", "create_deal": True },
            "900": { "msg": "Let it trade freely", "create_deal": True },
            "911": { "msg": "NEWS PENDING", "create_deal": True },
            "1000": { "msg": "Don't let it run!", "create_deal": True },
            "2100": { "msg": "Let it run!", "create_deal": True },
        }

        volume = volumes.get(deal['volume'])

        msg = volume.get('msg')
        direction = volume.get('direction', {}).get(deal['trdBs'], "")
        suffix = deal.get(volume.get("suffix"), "")

        console.print(f"{base_str}{msg}{direction}{suffix}")
        if volume.get('create_deal'):
            create_deal(deal)
```

If not, I may open a separate PR once this is merged.

Comment (mlebjerg) [reply mm_signals.py:117]: Thank you for helping review!
This looks really cool and easy to change, but it took me while to understand how it worked so i feel readability went down so it is a trade off, but I like it. Please do make your own pr after this is merged!

File: mm_graph.py (hunk 2 above)
Comment (mlebjerg) [reply mm_graph.py:22]: This is from shamelessly stolen code from a tutorial, we should delete or condense it down

File: mm_signals.py (hunk 3 above)
Comment (peacefulchickenk) [reply mm_signals.py:117]: Sounds good, I was worried about readability too with the special handling for `500` and `600`. I'll give it another run through for readability and see what I can do!

Comment (mlebjerg) [reply mm_signals.py:117]: Sounds good! I will set these as resolved, and I will wait for your PR!

File: mm_graph.py (hunk 2 above)
Comment (EONRaider) [reply mm_graph.py:22]: There's still a lot of ground to cover. I'll handle this one as soon as I solve that problem with the .conf file we discussed yesterday.

Comment (mlebjerg) [reply mm_graph.py:22]: Sounds good! I figured out the timeout problem, problem is in the the webull api code. It is set to timeout=1 in the the loop_forever(), changed it to 0, and the problem went away. Either i have to ask make a contribution there, it is something we have to fix on our end.

I also found a way to use webull's candles, that way we actually don't have to use yfinance, but it takes a little longer to get the data.

**Comment by mlebjerg** [comment]
I changed the branch to dev, that way i can merge this now, and we can work on it.

**Pull request merged**